tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -48,20 +53,29 @@ const mobileTopBarTitle = () => $(`${MOBILE_FILTER_TOP_BAR} .ellipsis-title`);
 
 //Hamburguer menu
 //User settings
-const USER_SETTINGS = '#header-dropdown a[routerlink="user"] i.fa-user';
+// const USER_SETTINGS = '#header-dropdown a[routerlink="user"] i.fa-user';
+// const userSettingsMenuOption = () => $('[test-id="user-settings-menu-option"]');
+const USER_SETTINGS = 'aria/User settings';
 const EDIT_PROFILE = '.user .configuration.page i.fa-user';
 // Feedback or Report bug
-const FEEDBACK_MENU = '#header-dropdown i.fa-bug';
+// const FEEDBACK_MENU = '#header-dropdown i.fa-bug';
+const FEEDBACK_MENU = 'aria/Report bug';
```

Comment:
```suggestion
const feedbackMenuOption = () => $('aria/Report bug');
```
Same here, it should be a method